packages/nvidia_nat_a365/tests/tooling/test_auth_integration.py

```
@@ -0,0 +1,700 @@
+                                                    mock_mcp_client_function_group):
+        """Test that string token doesn't pass auth to MCP servers."""
+        try:
+            from nat.plugins.mcp.client.client_config import MCPClientConfig
+        except ImportError:
+            pytest.skip("nvidia-nat-mcp not installed")
+
+        mock_a365_service.list_tool_servers.return_value = mock_mcp_servers
+
+        config = A365MCPToolingConfig(
+            agentic_app_id="test-agent",
+            auth_token="string-token-123",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)